tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es6",
+    "target": "es5",
```

Comment:
"ES5" supported by most browsers and is very stable. There are lot of compile time errors with es6 as typescript and es6(ES2015) needs to be handled very carefully. 

So I followed the angular official doc.
https://angular.io/docs/ts/latest/guide/typescript-configuration.html
Also with core-js instead of es6-shim, es5 seems to be much stable version than es6.